components/AboutUs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Aboutus is a bad component, it would be better to make this a generic `TextSection` component and handle all the content from the home section.

Reply:
changed